README.md

```diff
@@ -2,78 +2,67 @@
 
 Website:  https://www.operate-first.cloud/
 
-This is the support repository for the Operate First Initiative.
+GitHub Support Repository: https://github.com/operate-first/support
 
-For any questions, concerns, and/or requests, please use the appropriate [Issue Template][1]. If no template fits your requirement, then please make a regular GitHub issue [here][2].
+For any questions, concerns, and/or requests, please use the appropriate [Issue Template][1] to open an issue on our GitHub. If no template fits your requirement, then please make a regular GitHub issue [here][2].
+
+## End User Support
+
+We have a [community slack channel](https://join.slack.com/t/operatefirst/shared_invite/zt-o2gn4wn8-O39g7sthTAuPCvaCNRnLww) where we post announcements, general information, and more. If you have any questions, feel free to post a message to the `#support` channel. 
```

Comment:
@margarethaley 

> When I go into details it says that Trim Trailing Whitespace failed (not sure exactly what that means) 

Trailing whitespaces: 
> Trailing whitespace is any spaces or tabs after the last non-whitespace character on the line until the newline.

So if you see at the end of line 11, there is actually a blank space. If you remove it, then it should be fine.

Reply:
> I tried to run the tests locally according to those directions but am having trouble installing pre-commit. 

Feel free to reach out if you need help with this! Running the command will also apply some of these fixes automatically for you.

Reply:
Ohh okay! I'll fix that - thanks so much @HumairAK!